app/ui.js

```diff
@@ -11,14 +11,12 @@
 /* jslint white: false, browser: true */
 /* global window, document.getElementById, Util, WebUtil, RFB, Display */
 
-/* [module]
- * import Util from "../core/util";
- * import KeyTable from "../core/input/keysym";
- * import keysyms from "../core/input/keysymdef";
- * import RFB from "../core/rfb";
- * import Display from "../core/display";
- * import WebUtil from "./webutil";
- */
+import Util from "../core/util.js";
+import KeyTable from "../core/input/keysym.js";
+import keysyms from "../core/input/keysymdef.js";
+import RFB from "../core/rfb.js";
+import Display from "../core/display.js";
+import WebUtil from "./webutil.js";
```

Comment:
Got a reference? The ECMAScript spec is just horrible to read, but it doesn't seem to mention module name resolution at all. I found this:

https://github.com/whatwg/loader

But it just has a big **TODO** and examples so far.